Core/src/org/sleuthkit/autopsy/corecomponents/DataContentViewerHTML.java

```diff
@@ -0,0 +1,293 @@
+                    javaScriptButton.setText(disableJavaScriptTxt);
+                }
+                webEngine.reload();                
+            });
+            
+            btnBox.getChildren().add(javaScriptButton);
+            
+            //Webview components
+            browser = new WebView();
+            webEngine = browser.getEngine();
+            
+            //Javascript disabled by default
+            webEngine.setJavaScriptEnabled(false);
+            
+            root.setTop(btnBox);
+            root.setCenter(browser);
+            jfxPanel.setScene(scene);
+        });
+    }
+    
+    @Override
+    public void setNode(Node selectedNode) {
+
+        if ((selectedNode == null) || (!isSupported(selectedNode))) {
+            resetComponent();
+            return;
+        }
+
+        AbstractFile file = selectedNode.getLookup().lookup(AbstractFile.class);
+        if (file == null) {
+            resetComponent();
+            return;
+        }
+        
+        dataSource = file;
+        
+        resetComponent();
+
+        //Local files with .html extensions are loaded with a file URL
+        if(dataSource.getType() == TSK_DB_FILES_TYPE_ENUM.LOCAL &&
+                dataSource.getLocalAbsPath().endsWith(".html")) {
+            loadWebEngineFileURL(dataSource.getLocalAbsPath());
+        }
+        else { //Non-local files, or local files without a .html extension are loaded as a string
+            try {            
+
+                long size = file.getSize();
+                byte[] dataBytes = new byte[(int)size];
+                dataSource.read(dataBytes, 0, (int)size);
+                String HTML = new String(dataBytes);
+                loadWebEngineContent(HTML);
+
+            } catch (TskCoreException ex) {
+                Exceptions.printStackTrace(ex);
+            }
+        }
+        
+    }
+
+    @Override
+    public String getTitle() {
+        return NbBundle.getMessage(this.getClass(), "DataContentViewerHTML.title");
+    }
+
+    @Override
+    public String getToolTip() {
+        return NbBundle.getMessage(this.getClass(), "DataContentViewerHTML.toolTip");
+    }
+
+    @Override
+    public DataContentViewer createInstance() {
+        return new DataContentViewerHTML();
+    }
+
+    @Override
+    public Component getComponent() {
+        return this;
+    }
+
+    @Override
+    public void resetComponent() {
+        resetComponents();
+    }
+
+    @Override
+    public boolean isSupported(Node node) {
+        
+        if (node == null) {
+            return false;
+        }
+        
+        if(node.getLookup().lookupAll(BlackboardArtifact.class).size() > 0) {
+            return false;
+        }
+        
+        AbstractFile file = node.getLookup().lookup(AbstractFile.class);
+        
+        if (file != null && file.getSize() > 0 && file.isFile()) {
+            
+            String acceptedMIMEType = NbBundle.getMessage(this.getClass(), 
```

Comment:
why is this in the bundle?  Also we the file type detector is not 100% reliable and we fairly often get html files tagged at text/xml instead.  They are not going to be viewable.